octoprint_octorelay/__init__.py

```diff
@@ -153,7 +159,7 @@ def on_api_command(self, command, data):
         if command == GET_STATUS_COMMAND: # API command to get relay status
             return self.handle_get_status_command(data["pin"])
         if command == UPDATE_COMMAND: # API command to toggle the relay
-            return self.handle_update_command(data["pin"])
+            return self.handle_update_command(data)
```

Comment:
Let's call it `target`, since `cmd` is used for referring to system commands.

```suggestion
            return self.handle_update_command(data["pin"], data.get("target"))
```

Reply:
when optional this breaks, added a line to check if it exists first.